module/zfs/vdev_disk.c

```diff
@@ -824,7 +830,8 @@ vdev_disk_read_rootlabel(char *devpath, char *devid, nvlist_t **config)
 		/* read vdev label */
 		offset = vdev_label_offset(size, i, 0);
 		if (vdev_disk_physio(bdev, (caddr_t)label,
-		    VDEV_SKIP_SIZE + VDEV_PHYS_SIZE, offset, READ_SYNC) != 0)
+		    VDEV_SKIP_SIZE + VDEV_PHYS_SIZE, offset, READ,
+		    REQ_SYNC) != 0)
```

Comment:
REQ_SYNC does not exist in 2.6.32 kernels, however, since this function isn't compiled under Linux any more, it won't matter.